doc/dune

```diff
@@ -10,23 +12,33 @@
   ;   + tools/coqdoc/coqdoc.css
   (package coq)
   (source_tree sphinx)
-  (source_tree tools)
+  (source_tree tools/coqrst)
   unreleased.rst
   (env_var SPHINXWARNOPT))
  (action
-  (run env COQLIB=%{project_root} sphinx-build %{env:SPHINXWARNOPT=-W} -b html -d sphinx_build/doctrees sphinx sphinx_build/html)))
+  (run env COQLIB=%{project_root} sphinx-build %{env:SPHINXWARNOPT=-W} -b html sphinx %{targets})))
 
-(alias
- (name refman-html)
- (deps sphinx_build))
+(rule
+ (targets refman-pdf)
+ (alias refman-pdf)
+ (package coq-doc)
+ (deps
+  (package coq)
+  (source_tree sphinx)
+  (source_tree tools/coqrst)
+  unreleased.rst
+  (env_var SPHINXWARNOPT))
+ (action
+  (progn
+   (run env COQLIB=%{project_root} sphinx-build %{env:SPHINXWARNOPT=-W} -b latex sphinx %{targets})
+   (chdir %{targets} (run make)))))
 
 (rule
  (targets unreleased.rst)
  (deps (source_tree changelog))
  (action (with-stdout-to %{targets} (bash "cat changelog/00-title.rst changelog/*/*.rst"))))
 
-; The install target still needs more work.
-; (install
-;  (section doc)
-;  (package coq-refman)
-;  (files sphinx_build))
+(install
```

Comment:
Unfortunately this is broken [as you have witnessed] and still not fixed. The corresponding upstream issue is https://github.com/ocaml/dune/issues/1377 , see also https://github.com/coq/coq/pull/9636 and https://github.com/ocaml/dune/issues/1868

Reply:
I see, thanks for the links! But it seems that it is not just an issue of "installing directory is not supported" but also that Dune doesn't realize that this is unsupported and fails pretty badly when trying anyway.

Reply:
Yes I agree; the solution to this is a bit tricky unfortunately. Some ideas are:
- generate the install file using a small program [but that's tricky to maintain]
- use a small makefile to install the docs [that's what dune does, well, in this case to deploy the sphinx manual as it is not distributed in the opam package I think]